### DISCUSSION TITLE
How/where to install additional bash tools?

### DISCUSSION THREAD

**Discussion by asdf23** [vercel/hyper#5998] [Q&A] [answered]
Sorry if this question is strictly unrelated to Hyper..
On windows running bash... how do I install programs such as zip or other tools I would normally dnf/yum/apt-get install?

**Comment by LabhanshAgrawal** [comment, marked as answer]
yeah, it is unrelated to Hyper
if you are using wsl then it's the same as using linux so you can use the package manager from that
don't know about git bash though if you are using that
